provider/oci/images.go

```diff
@@ -39,6 +40,8 @@ const (
 	centOS   = "CentOS"
 	ubuntuOS = "Canonical Ubuntu"
 
+	UbuntuBase = "ubuntu"
```

Comment:
isn't this a const somewhere already?

Reply:
there might be but I haven't found one, I tried searching many different regexes and nothing came up. We have ~17000 occurrences of "ubuntu" so it might be there somewhere :man_shrugging:

Reply:
I eventually found one, but it's defined with "Ubuntu". Juju doesn't care about capitalization, but some pieces do, let's leave this.